components/02-form-elements/typeahead/typeahead.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="field">
+   <div class="typeahead js-typeahead" data-content="{{json content}}" data-result-limit="{{result_limit}}">
+    {{> @label }}
+    <select class="typeahead__select js-typeahead-select input input--select{{#if select_classes}}{{select_classes}}{{/if}}" id="{{label_for}}" name="select">
+      {{#each options}}
+        <option{{#if option_value}} value="{{option_value}}"{{/if}}{{#if selected}} selected{{/if}}{{#if disabled}} disabled{{/if}}{{#if option_alternatives}}data-alternatives=" {{option_alternatives}}"{{/if}}>{{option_text}}</option>
```

Comment:
### Ethnicity
`value="{{option_value}}"` appears to populate with spaces in the value. For example: `value=" Welsh"`

Reply:
This and the comment above about `French` and `Polynesian` should have been resolved from removing the leading spaces from the ethnicities data.